docs/data/stashcache/install-cache.md

```diff
@@ -4,6 +4,10 @@ DateReviewed: 2023-03-03
 Installing the OSDF Cache
 =========================
 
+!!! warning "Deprecation warning"
+    This document is outdated and describes an XCache-based OSDF Cache install, which is deprecated.
+    Future OSDF Caches should be based on Pelican; documentation for a Pelican-based OSDF Cache install is forthcoming.
```

Comment:
XCache based caches are deprecated, pelican is the future, but we have no docs on installing pelican caches...

Reply:
Yeah, we're working on it. In the meantime, though, we'd rather not have people setting up new XCaches...